### PR TITLE
Add legacy role filter for tag(s) & reduce default page size to 10

### PR DESCRIPTION
**Pull request by jctanner**
https://issues.redhat.com/browse/AAH-2154
https://issues.redhat.com/browse/AAH-2085

### CODE REVIEW

File: galaxy_ng/tests/integration/api/test_community.py [hunk 1]
```diff
@@ -524,3 +524,62 @@ def get_roles(page_size=1, order_by='created'):
 
     # cleanup
     clean_all_roles(ansible_config)
+
+
+@pytest.mark.community_only
+def test_v1_role_tag_filter(ansible_config):
+    """" Tests if v1 roles are auto-sorted by created """
+
+    config = ansible_config("admin")
```
Comment (awcrosby) [inline galaxy_ng/tests/integration/api/test_community.py:533]: if just testing search, can probably test with a non admin profile

File: galaxy_ng/app/api/v1/filtersets.py [hunk 2]
```diff
@@ -57,6 +57,8 @@ class LegacyRoleFilter(filterset.FilterSet):
 
     github_user = filters.CharFilter(method='github_user_filter')
     keywords = filters.CharFilter(method='keywords_filter')
+    tags = filters.CharFilter(method='tags_filter')
+    tag = filters.CharFilter(method='tags_filter')
```
Comment (awcrosby) [inline galaxy_ng/app/api/v1/filtersets.py:61]: i see in the integration test that `tag` is a query param... is `tags` used by the UI also?

Comment (jctanner) [reply galaxy_ng/app/api/v1/filtersets.py:61]: The UI uses "tags", but it doesn't make contextual sense if you are filtering on a single tag. We could extend "tags" to mean a comma separated list, but there's no convention or ask for that yet.

File: galaxy_ng/tests/integration/api/test_community.py (hunk 1 above)
Comment (jctanner) [reply galaxy_ng/tests/integration/api/test_community.py:533]: Sync requires an admin to run.

**Review by awcrosby — APPROVED** [review]

**Pull request merged**